src/assetPath.js

```diff
@@ -0,0 +1,5 @@
+global.__ml_playground_asset_public_path__ = './assets/';
+
+export const setAssetPath = (path) => {
+  global.__ml_playground_asset_public_path__  = path;
+};
```

Comment:
Is this asset path to access the the datasets? I'm not clear on how this part works.

Reply:
Yes, this variable is used to override the asset path by the main repo's code [here](https://github.com/code-dot-org/code-dot-org/pull/37495/files#diff-f2ae9cbcf69d10020839a1ef20c80985272ddde9ed3baf55d36bf532229fe8b6R122). 

If we want to publish assets to be imported by the app we can also use it like this: https://github.com/code-dot-org/ml-activities/blob/main/webpack.config.js#L49.

One subtle caveat is that the variable is reset [here](https://github.com/code-dot-org/ml-playground/pull/25/files#diff-64698619a0c48c473fa8e31b2ebe38f6ac73bbeb7f8a491dd65aed5d67b79579R1) if we `require` this module, say at app load time, so we need to be careful about its usage.